src/dm/dm_network.cpp

```diff
@@ -176,7 +176,7 @@ void dm_network_t::operator = (const dm_network_t& obj)
 {
     memcpy(&this->m_net_info.id,&obj.m_net_info.id,sizeof(em_long_string_t));
     this->m_net_info.num_of_devices = obj.m_net_info.num_of_devices;
-    strncpy(this->m_net_info.timestamp, obj.m_net_info.timestamp, strlen(obj.m_net_info.timestamp) + 1);
+    memmove(this->m_net_info.timestamp, obj.m_net_info.timestamp, strlen(obj.m_net_info.timestamp) + 1);
```

Comment:
Similar to the previous `memmove` suggestion
```suggestion
    memcpy(this->m_net_info.timestamp, obj.m_net_info.timestamp, sizeof(em_long_string_t));
```